hugegraph-llm/src/hugegraph_llm/api/vector_api.py

```
@@ -0,0 +1,77 @@
+from hugegraph_llm.api.models.rag_requests import GraphConfigRequest
+from hugegraph_llm.config import huge_settings
+
+API_CALL_TRACKER = {}
+
+
+# pylint: disable=too-many-statements
+def vector_http_api(router: APIRouter, update_embedding_func):
+    @router.post("/vector/embedding", status_code=status.HTTP_200_OK)
+    def update_embedding_api(
+            daily_limit: int = 50,
+            graph_config: Optional[GraphConfigRequest] = Body(None)
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)